connect-examples/v2/php_checkout/README.md

```diff
@@ -1,49 +1,49 @@
 # Useful Links
 
-* [PHP SDK Page](https://developer.squareup.com/docs/sdks/php)
-* [Checkout API Overview](https://developer.squareup.com/docs/checkout-api/what-it-does)
-* [Checkout in the API Reference](https://developer.squareup.com/reference/square/checkout-api)
+- [PHP SDK Page](https://developer.squareup.com/docs/sdks/php)
+- [Checkout API Overview](https://developer.squareup.com/docs/checkout-api/what-it-does)
+- [Checkout in the API Reference](https://developer.squareup.com/reference/square/checkout-api)
 
-Square Checkout Demo
-=========================
+# Square Checkout Demo
 
 This is a simple example application that utilizes Square's Checkout API. This examples does assume you are familiar with PHP development.
 
 It takes a single payment, declared by the user, and creates an order to use in the Checkout API.
-## Setup
 
+## Setup
 
 ### Requirements
 
-* 7.1 <= PHP
+- 8.0 <= PHP
 
 ### Install the PHP client library
 
 This sample already includes the `square/square` dependency in its `composer.json`
 file. To install the client library:
 
 1. Make sure you've downloaded Composer, following the instructions
-[here](https://getcomposer.org/download/).
+   [here](https://getcomposer.org/download/).
 
 2. Run the following command from the directory containing `composer.json`:
 
 ```
-composer install
+php composer.phar install
```

Comment:
Is `php` a syntax highlighting hint? If so, I think it should be on the previous line instead.